installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3187,3 +3187,5 @@ END //
 DELIMITER ;
 
 CALL change_createdTime_columntype_to_timestamp();
+
+update cf_Target set targetConfig='{"key":"id,databaseVersion","id":"id"}'  where dataSourceName="gcp" and targetName in ('cloudsql_mysqlserver','cloudsql_postgres','cloudsql_sqlserver');
```

Comment:
since this query will be executed anyway, do we really need those changes above?

Reply:
It can avoid confusion when read. Also makes sense to remove incorrect values.

Reply:
this whole file is not meant to be read.
from data stand point - are those changes above adding anything what is not done by this exact line?

Reply:
No, strictly from data point of view, update is enough.